xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -803,11 +828,11 @@ CGUIFontTTF::Character* CGUIFontTTF::GetCharacter(character_t chr, FT_UInt glyph
 }
 
 bool CGUIFontTTF::CacheCharacter(wchar_t letter, uint32_t style, Character* ch, FT_UInt glyphIndex)
-{
+{;
   if (!glyphIndex)
     glyphIndex = FT_Get_Char_Index(m_face, letter);
 
-  FT_Glyph glyph = NULL;
+  FT_Glyph ftGlyph = NULL;
```

Comment:
Is this variable name change is really needed inside function as we are only dealing with freetype inside `CacheCharacter`?

Reply:
Originally I planned to save the glyph info with the character, then I saw that the cached character shouldn't save positions because it'll be outdated values, which introduced bugs and fixed it later, and didn't make a refactoring.